test/distributed/tensor/parallel/test_tp_examples.py

```diff
@@ -122,7 +135,9 @@ def _test_mlp_inference(self, device_mesh):
 
     @with_comms
     @parametrize("is_seq_parallel", [True, False])
+    # TODO: need to revisit input_reshard API about why it failed multi-gpu tests.
     @parametrize("recompute_activation", [True, False])
```

Comment:
did we disable the `recompute_activation` option?

Reply:
It doesn't fail the test here. It only breaks the test in `test_fsdp_2d_parallel.py`. So I kept it here. Shall we disable it altogether?

Reply:
disabled altogether